internal/credentials/file.go

```diff
@@ -207,6 +209,50 @@ func (c *fileCredentialsService) Delete(guid string) error {
 	return nil
 }
 
+// Resets the credentials file
+// it is a last resort in case the data turns out to be irrecognizable
+// Returns the backup path of the original credentials file
+func (c *fileCredentialsService) Reset() (string, error) {
+	copyFilename, err := c.backupFile()
+	if err != nil {
+		return "", err
```

Comment:
Have we looked at what the user experience will be if we are unable to backup the file (with the raw errors able to be returned from the file operations within `.backupFile()`)? I would think we might want to capture these a bit and convert them into a nicer message for the user.

Reply:
What do you think about handling this as a follow-up?

Reply:
We could certainly do it in a separate PR, but I think we should implement it prior to releasing it.